superset-frontend/src/components/ListView/CardSortSelect.tsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useState } from 'react';
+import { styled, withTheme, SupersetThemeProps } from '@superset-ui/style';
+import { PartialThemeConfig, Select } from 'src/components/Select';
```

Comment:
we probably could have gotten away with an [antd dropdown](https://ant.design/components/dropdown/) as we'll only be rendering a few items and don't need search, but this is good for consistency with the filter components.